js/language_server/Makefile

```diff
@@ -23,11 +33,25 @@ package-lock.json:
 node_modules/.package-lock.json: package-lock.json
 	npm ci
 
+../tree-sitter/dist/tree-sitter.o:
+	$(MAKE) -C ../tree-sitter dist/tree-sitter.o
+
+dist/language-server-c-bindings.js: ../libyaml/dist/libyaml.o ../libpcre/dist/libpcre.a ../tree-sitter/dist/tree-sitter.o $(LANGUAGES:%=../languages/%/dist/semgrep-parser.o)
+	mkdir -p dist
+	emcc \
+		-O3 \
+		$^ \
+		$(EMCC_DEFAULTS) \
+		-s INITIAL_MEMORY=31457280 \
+		-s SINGLE_FILE=1 \
+		-sEXPORTED_FUNCTIONS=_malloc,_free,$(YAML_EXPORTED_FUNCTIONS),$(PCRE_EXPORTED_FUNCTIONS),$(TREESITTER_EXPORTED_FUNCTIONS),$(PARSER_EXPORTED_FUNCTIONS) \
+		-o $(word 1,$@)
```

Comment:
you can replace `$(word 1,$@)` with `$@` -- the `word` was needed when this had multiple targets